### PR TITLE
MODIFY Image.py save() -> ADD convert JPG->JPEG code

### PR DESCRIPTION
**Pull request by ccgeonhee**
Fixes # .

Changes proposed in this pull request: If the extension is JPG, there is an error that cannot be saved. Therefore, I added a code that converts JPG to JPEG. You can add to FORMAT_MAP for similar cases. For example, MPG to MPEG.

 * Convert JPG to JPEG

Thank you.

### CODE REVIEW

File: src/PIL/Image.py [hunk 1]
```diff
@@ -97,7 +97,7 @@ class DecompressionBombError(Exception):
     # import Image and use the Image.core variable instead.
     # Also note that Image.core is not a publicly documented interface,
     # and should be considered private and subject to change.
-    from . import _imaging as core
+    from .. import _imaging as core
```
Comment (radarhere) [inline src/PIL/Image.py:100]: This change isn't related to your initial description. Was it unintentional?

Comment (ccgeonhee) [reply src/PIL/Image.py:100]: Oh,Sorry. I didn't delete it after the test.

Comment (radarhere) [reply src/PIL/Image.py]: Thanks

**Comment by radarhere** [comment]
Just to be clear - this should only be helpful to users who are saving an image to a file object instead of a filename, or who are saving to a file but don't want the extension derived from the filename.
```python
from PIL import Image
import io
im = Image.new("RGB", (1, 1))
fp = io.BytesIO()
im.save(fp, "JPG")
im.save("out", "JPG")
im.save("out.gif", "JPG")
```
I think most of the time, users just use a filename. The following code works fine.
```python
from PIL import Image
im = Image.new("RGB", (1, 1))
im.save("out.jpg")
```
If you are creating this PR because you would like the following to work -
```python
from PIL import Image
im = Image.new("RGB", (1, 1))
im.save("out.jpg", "JPG")
```
then I want to clarify that `format` could just be removed. It may seem obvious that it is optional, but I suspect not everyone realises this.

**Comment by ccgeonhee** [comment]
> Just to be clear - this should only be helpful to users who are saving an image to a file object instead of a filename, or who are saving to a file but don't want the extension derived from the filename.
> 
> ```python
> from PIL import Image
> import io
> im = Image.new("RGB", (1, 1))
> fp = io.BytesIO()
> im.save(fp, "JPG")
> im.save("out", "JPG")
> im.save("out.gif", "JPG")
> ```
> 
> I think most of the time, users just use a filename. The following code works fine.
> 
> ```python
> from PIL import Image
> im = Image.new("RGB", (1, 1))
> im.save("out.jpg")
> ```
> 
> If you are creating this PR because you would like the following to work -
> 
> ```python
> from PIL import Image
> im = Image.new("RGB", (1, 1))
> im.save("out.jpg", "JPG")
> ```
> 
> then I want to clarify that `format` could just be removed. It may seem obvious that it is optional, but I suspect not everyone realises this.

Oh, I see. I think I forgot it was an option. 
The error that occurs when the `format`exists is annoying, so I created pr. 
If you're going to get rid of the `format`, you don't need this pr.

**Comment by radarhere** [comment]
Oh no, I didn't mean that the `format` option will be removed from Pillow. I meant that users don't have to specify it if they don't want to.

**Comment by ccgeonhee** [comment]
Oh, haha I see.

**Comment by radarhere** [comment]
If there isn't a clear demand for this, then my preference is to keep the API simple.

**Pull request closed without merging**